docs/walkthrough.md

```diff
@@ -15,23 +15,23 @@ of this repository.
 
 # Step 1 - Installing the UPS Broker Server
 
-Since the Service Catalog provides a Kubernetes-native interface to an 
+Since the Service Catalog provides a Kubernetes-native interface to an
 [Open Service Broker API](https://www.openservicebrokerapi.org/) compatible broker
 server, we'll need to install one in order to proceed with a demo.
 
-In this repository, there's a simple, "dummy" server called the User Provided 
+In this repository, there's a simple, "dummy" server called the User Provided
 Service (UPS) broker. The codebase for that broker is
 [here](https://github.com/kubernetes-incubator/service-catalog/tree/master/contrib/pkg/broker/user_provided/controller).
 
-We're going to deploy the UPS broker to our Kubernetes cluster before 
-proceeding, and we'll do so with the UPS helm chart. You can find details about 
-that chart in the chart's 
+We're going to deploy the UPS broker to our Kubernetes cluster before
+proceeding, and we'll do so with the UPS helm chart. You can find details about
+that chart in the chart's
 [README](https://github.com/kubernetes-incubator/service-catalog/blob/master/charts/ups-broker/README.md).
 
 Otherwise, to install with sensible defaults, run the following command:
 
-```
-helm install charts/ups-broker --name ups-broker --namespace ups-broker
+```console
+helm install svc-cat/ups-broker --name ups-broker --namespace ups-broker
```

Comment:
Why this change? Aren't the charts in the `charts/` folder any more?

Reply:
@luksa Ah this command sensitive to where you run it; ok. Can I change this to `helm install ./charts/ups-broker...` so its clear its a local path rather than referencing the hosted `svc-cat/ups-broker` chart?